pom.xml

```diff
@@ -69,6 +69,10 @@
   </dependencyManagement>
 
   <dependencies>
+    <dependency>
+      <groupId>io.jenkins.plugins</groupId>
+      <artifactId>commons-lang3-api</artifactId>
+    </dependency>
```

Comment:
Would prefer not to add this dep unless it is really offering some nontirivial function it would be onerous to do without.

Reply:
removed